components/filecoin/rs/Cargo.toml

```diff
@@ -7,16 +7,18 @@ publish = false
 
 [dependencies]
 base64 = "0.13.0"
-blake2b_simd = "1.0"
 bls-signatures = { version = "0.12", default-features = false, features = ["pairing"] }
 cxx = { version = "1.0", features = [ "alloc", "std" ] }
-fvm_shared = { version = "0.7.0 " }
+fvm_shared = { version = "3.3.0 " }
 libsecp256k1 = "0.7"
 num_bigint_chainsafe = { package = "forest_bigint", version = "0.1.2"}
 serde = { version = "^1.0.117", features = ["derive"] }
 serde_json = "^1.0.59"
 thiserror = "^1.0.30"
-fvm_ipld_encoding = "0.2.0"
+blake2b_simd = "^1.0.0"
```

Comment:
if a more specific minimum version is not required, please default to the epoch `blake2b_simd = "1"`